src/TMainConsole.cpp

```diff
@@ -1145,7 +1145,6 @@ bool TMainConsole::setTextFormat(const QString& name, const QColor& fgColor, con
 
 void TMainConsole::printOnDisplay(std::string& incomingSocketData, const bool isFromServer)
 {
-    Q_ASSERT_X(mpLineEdit_networkLatency, "TMainConsole::printOnDisplay(...)", "mpLineEdit_networkLatency does not point to a valid QLineEdit");
```

Comment:
Are you should you want to take this out of ***debug*** builds?

Reply:
Fixed, bad merge.